docker-clean.sh

```diff
@@ -0,0 +1 @@
+sudo docker system prune -a
```

Comment:
hm I am not sure I want this to be an x1plus script.  this has far wider reaching consequences than just x1plus!